From 0f34459fcef6fade5811c5efa9eacae2898f343e Mon Sep 17 00:00:00 2001
From: Fabian Homborg <FHomborg@gmail.com>
Date: Sun, 1 Mar 2020 17:03:15 +0100
Subject: [PATCH] Disable svn prompt by default

This is apparently quite slow on large svn repos (like 40 seconds
slow), and we don't have a good thing to display other than the full
file information.

So we'll have to disable it for now.

Fixes #6681.

[ci skip]
---
 share/functions/fish_vcs_prompt.fish | 5 ++++-
 1 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/share/functions/fish_vcs_prompt.fish b/share/functions/fish_vcs_prompt.fish
index 5c1ca3d4350ff6238439d7932777bb2f83b09ed3..d0deef7c2796b545c1201b3eabd1584fa344a175 100644
--- a/share/functions/fish_vcs_prompt.fish
+++ b/share/functions/fish_vcs_prompt.fish
@@ -3,5 +3,8 @@ function fish_vcs_prompt --description "Print the prompts for all available vcse
     # This is so we don't try svn if git already worked.
     fish_git_prompt
     or fish_hg_prompt
-    or fish_svn_prompt
+    # The svn prompt is disabled by default because it's quite slow on common svn repositories.
+    # To enable it uncomment it.
+    # You can also only use it in specific directories by checking $PWD.
+    # or fish_svn_prompt
 end
